pkg/config/config.go

```diff
@@ -4,22 +4,28 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/cruise-automation/daytona/pkg/logging"
 )
 
+// Auth methods
+const (
+	K8s = "K8S"
+	AWS = "AWS"
+	GCP = "GCP"
+)
+
```

Comment:
I find custom types sometimes make the context easier to understand later on when being referenced
```
type AuthMethod string

const (
	K8S   AuthMethod   = "K8S"
)
```

or

```
const (
	AuthMethodK8s   AuthMethod   = "K8S"
)
```